sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/AdtQuerySelectFrom.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// Query that contains a SELECT and FROM clause.
+    /// </summary>
+    internal class AdtQuerySelectFrom : AdtQuery
+    {
+        /// <summary>
+        /// Adds a WHERE and its conditional argument(s) clause to the query object. Meant to be used for simple
+        /// conditions involving operators or with basic ADT functions.
+        /// </summary>
+        /// <param name="condition"> A custom object that encodes the logical statement nested within the WHERE clause. </param>
+        /// <returns> The ADTQuery object itself. </returns>
+        internal AdtQuery Where(BaseCondition condition)
+        {
+            Console.WriteLine(condition);
+            return this;
+        }
+
+        /// <summary>
+        /// An alternative way to add a WHERE clause to the query by directly providing a string that contains the condition.
+        /// </summary>
+        /// <param name="condition"> The verbatum condition (SQL-like syntax) in string format. </param>
+        /// <returns> The ADTQuery object itself. </returns>
+        public AdtQuery Where(string condition)
```

Comment:
it may be valuable to mention that all where clauses will be appended using a `AND` Operator.